contractTest/gateways/MsSQL/verifyUserLogin.contractTest.js

```diff
@@ -0,0 +1,68 @@
+import verifyUserLogin from "../../../src/gateways/MsSQL/verifyUserLogin";
+import AppContainer from "../../../src/containers/AppContainer";
+import setupUser from "../../../test/testUtils/setupUser";
+
+describe("verifyUserLogin contract tests", () => {
+  const container = AppContainer.getInstance();
+
+  xit("verifies if a admin code and password match an existing trust", async () => {
+    // await setupUser(container)({
+    //   email: "test@email.com",
+    //   password: "TESTPASSWORD",
+    //   type: "admin",
+    // });
```

Comment:
Prob want to remove this if not needed

Reply:
This is required to run the test, so has been uncommented.